scout/build/panel.py

```diff
@@ -70,6 +69,9 @@ def build_gene(gene_info, adapter):
     if gene_info.get("transcripts"):
         gene_obj["disease_associated_transcripts"] = gene_info["transcripts"]
 
+    if gene_info.get("comment"):
```

Comment:
Yeah, considered it, but only a couple are identical, so it would be like a dict constant with the names and types etc. It would lower the conceptual cognitive complexity, but I was unconvinced about the actual cognitive effort. But, if you also thought about it maybe I just do it.

At one point they developed this nice abstraction called object orientation, where data structures could have methods etc. Now that would solve it. 😅

Reply:
Of course I noticed! 😈